people.md

```diff
@@ -0,0 +1,47 @@
+# Steering Council and Institutional Partners
+
+Benevolent Dictator for Life
+============================
+
+Fernando Perez is the Benevolent Dictator for Life (BDFL).
+
+Steering Council
+================
+
+- Brian Granger
+- Damian Avila
+- Fernando Perez
+- Jason Grout
+- Jessica Hamrick
+- Jonathan Frederic
+- Kyle Kelley
+- Matthias Bussionnier
+- Min Ragan-Kelley
+- Thomas Kluyver
+
+NumFOCUS Subcommittee
```

Comment:
I am fine with the original name. Based on discussion that are happening in NumFOCUS the structure of these teams and relationship to project leadership and governance may change over time - but we aren't there yet.